src/instance.jl

```diff
@@ -89,16 +93,18 @@ function MOI.delete!(m::AbstractInstance, vr::MOI.VariableReference)
     for cr in rm
         MOI.delete!(m, cr)
     end
+    splice!(m.varrefs, vf)
```

Comment:
The second argument of `splice!` should be the indices. What is `vf` ? I think `delete!(m.varrefs, vr)` would work if `vr` is a `Set`.